x-pack/plugins/cases/common/api/cases/user_actions/common.ts

```diff
@@ -51,15 +51,12 @@ export const UserActionCommonAttributesRt = rt.type({
  * This should only be used for the getAll route and it should be removed when the route is removed
  * @deprecated use CaseUserActionInjectedIdsRt instead
  */
-export const CaseUserActionSavedObjectIdsRt = rt.type({
+export const CaseUserActionInjectedDeprecatedIdsRt = rt.type({
   action_id: rt.string,
   case_id: rt.string,
   comment_id: rt.union([rt.string, rt.null]),
 });
 
-export type UserActionWithAttributes<T> = T & rt.TypeOf<typeof UserActionCommonAttributesRt>;
-export type UserActionWithResponse<T> = T & rt.TypeOf<typeof CaseUserActionSavedObjectIdsRt>;
-
```

Comment:
I can't comment on the exact line (60) https://github.com/elastic/kibana/blob/1546af64941b65e12534788d946b9ea4faa11be1/x-pack/plugins/cases/common/api/cases/user_actions/common.ts#L60-L63

Let's remove that comment.